primitives/src/pcs/univariate_kzg/mod.rs

```diff
@@ -848,12 +843,11 @@ pub(crate) mod icicle {
                 ));
             }
 
-            let size = polys[0].degree() + 1;
-            if polys.iter().any(|p| p.degree() + 1 != size) {
-                return Err(PCSError::InvalidParameters(
-                    "all polys should have the same degree".to_string(),
-                ));
-            }
+            let size = polys
```

Comment:
Maybe give a more informative variable name here. Also what's the reason that we need to set it to max_degree + 1?

Reply:
done in 997e2e1